scripts/collect_steps.py

```diff
@@ -19,7 +19,7 @@
     "--num-multi-bits",
     "3",
 ]
-QUERY_SIZE = 10
+QUERY_SIZE = 100
```

Comment:
@akoshelev - this line is the key. If you want to run for a larger number of input rows, you need to update this, then regenerate the steps. This is due to the "interaction pattern" stuff which is a binary tree. So there are log(n) steps for various things (like attribution). If you want to run on 20 million rows, you'll need this number to be a lot higher to generate more than "depth=6" which is what the steps currently have. That should support ~2^6 rows, but you'll need a lot more.

Reply:
is it 2^6 per user or overall that depth=6 will support?

Reply:
For old IPA, the steps depend upon the total number of rows (and not upon the number of rows per user).

That's no longer the case for OPRF-IPA, but the config to generate those steps is lower down in this file.

Reply:
Interesting - presumably we have been running the step generation process with 10 rows for a while. It was working fine for up to 20M rows.